test/test-lib/api-helpers.ts

```diff
@@ -1,9 +1,10 @@
 import { expect } from './chai';
-import supertest = require('./supertest');
+import supertest from './supertest';
 
 import { app } from '../../init';
 import { SUPERUSER_EMAIL, SUPERUSER_PASSWORD } from '../../src/lib/config';
 import { User } from '../../src/platform/jwt';
+export { User as Admin } from '../../src/platform/jwt';
```

Comment:
What's the benefit of exporting this as `Admin` instead of just re-exporting the `User`?

NIT: How about
```
import { User as Admin } from '../../src/platform/jwt';
```
so the return type is consistent w/ the `let admin: Admin;` that we have?

Reply:
Changed it to `User`, I'm not convinced anymore `Admin` provides any value -- admins are users after all.

Reply:
Changed it to `User`, I'm not convinced anymore `Admin` provides any value -- admins are users after all.